ocrspace/main.py

```diff
@@ -28,25 +29,45 @@ class Language:
     Turkish = 'tur'
 
 
+@unique
+class Engine(Enum):
```

Comment:
Use `IntEnum` instead; that way you can directly compare with an integer and it will still work automatically

Reply:
resolved on c9ffd78ddc93ede49af145fe706ee269c4e0a42c